docs/configuration_files.md

```diff
@@ -720,6 +724,103 @@ This task will fetch current pokemon spawns from /raw_data of an PokemonGo-Map i
 }
 ```
 
+## Sniping _(Sniper)_
+[[back to top](#table-of-contents)]
+
+### Description
+[[back to top](#table-of-contents)]
+
+This task is an upgrade version of the MoveToMapPokemon task. It will fetch pokemon informations from any number of urls (sources), including PokemonGo-Map, or from the social feature. You can also use the old PokemonGo-Map project. For information on how to properly setup PokemonGo-Map have a look at the Github page of the project [here](https://github.com/PokemonGoMap/PokemonGo-Map). You can also use [this](https://github.com/YvesHenri/PogoLocationFeeder), which is an adapted version of the application that NecroBot used to snipe. There is an example config in `config/config.json.map.example`.
+
+### Options
+[[back to top](#table-of-contents)]
+
+* `mode` - The mode on which the sniper will fetch the informations. (default: social)
+   - `social` - Information will come from the social network.
+   - `url` - Information will come from one or multiple urls.
+* `bullets` - Each bullet corresponds to an attempt of catching a pokemon. (default: 1)
```

Comment:
I would add an example here. I still don't understand what that means.
i understand that I will try to catch it only once (unlikely that this is its meaning :) ), and next option ensures that I won't miss.